### PR TITLE
Update test rules user guide

### PR DESCRIPTION
**Pull request by J-Binieda-ericsson**
### Applicable Issues
Test Rules GUI has been updated under this sprint and documentation on Github is outdated.

### Description of the Change
* I rewrote documentation for Test Rules GUI.
* I moved all images that are used in the documenatation to the wiki folder.
* Updated links in EI front end.

### Benefits
Updated documentation

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by: Jakub Binieda, jakub.binieda@ericsson.com

### CODE REVIEW

File: wiki/Test-Rules.md [hunk 1]
```diff
@@ -1,657 +1,258 @@
...
-</pre>
+<h3>Find Aggregated Object</h3>
+
+<p>Clicking on "Find Aggregated Object" button will start the aggregation process. If rules and events are correct, a pop-up window with the aggregated object will show up on the screen.</p>
+
+<img src="./images/GUI_TestRules_Aggregated_Object.png"></img>
 
-<p>Also note that if you refer to a key that does not exist, a value of null (or the language equivalent of null) is
-    returned.</p>
+**_More information about how to write rules can be found [here](https://github.com/eiffel-community/eiffel-intelligence)._**
```
Comment (Christoffer-Cortes) [inline wiki/Test-Rules.md]: This links to the ei back end repo. Is there a more specific link? I found wiki docs Rules.md
 and TestRulesUserGuide.md, the latter which I'm not sure if it's supposed to be there.

Comment (J-Binieda-ericsson) [reply wiki/Test-Rules.md]: I could change this link to the more specific link for example to Rules.md page. I though that in main page has better overview of all rules sections, but is maybe better to link to one specific.

Comment (evasiba-ericsson) [reply wiki/Test-Rules.md]: Yes, a link to the rules documentation in backend is better that this general link.

Comment (J-Binieda-ericsson) [reply wiki/Test-Rules.md]: Done

**Pull request merged**